packages/neutrino/src/api.js

```diff
@@ -26,14 +27,28 @@ const pathOptions = [
 
 // getOptions :: Object? -> IO Object
 const getOptions = (opts = {}) => {
+  let fileExtensions = new Set(['js', 'jsx', 'vue', 'ts', 'mjs']);
```

Comment:
Let's rename this to `moduleExtensions` so it's clear we are only dealing with module code instead of things like images and stylesheets.